submissions/tests/test_team_api.py

```diff
@@ -195,23 +195,42 @@ def test_create_submission_for_team_existing_individual_submission(self):
         """
         Test for calling create_submission_for_team when a user somehow already has
         an existing active submission for the item.
-
-        For normal Submissions, if a submission exists and we create another one, the second
-        will increment the first's attempt_number. However, for team submissions, we currently
-        pass the attempt_number from team_api.create_submission to api.create_submission, so
-        all created individual submissions will have the same attempt_number
+        This can happen if a user was on a team that submitted and then is reassigned to a team
+        that didn't have a submission and is submitting.
+        Expected outcome: submission is created because the previous submission was not team based
```

Comment:
This is probably good to test just for error behavior, but let's make it clear that this would require staff to release an ORA as an individual ORA, a user to submit, then for staff to change to ORA to team based and then student to submit again.

Reply:
Haha wait I wrote this test lmao

Reply:
This is kinda a silly test in retrospect, but hey, good on ya for not just deleting it :p

Reply:
Ask and you shall receive :)